config-common/Cargo.toml

```diff
@@ -6,6 +6,10 @@ edition = "2018"
 
 
 [dependencies]
+async-trait = "0.1"
 backtrace = "0.3"
+futures-util = "0.3"
+notify = "4"
 serde = "1"
+tokio = { version = "1", features = ["rt"] }
```

Comment:
Also need the `"sync"` feature for `tokio::sync::mpsc`

Reply:
wait, then how did this compile in the first place?

Reply:
I was able to compile without "sync"

Reply:
Features get unioned across the workspaces, and something else is enabling it.

I didn't say it wouldn't compile. I said it's needed by the code. Hence it needs to be listed.

Reply:
Fixed